app/helpers/samples_helper.rb

```diff
@@ -352,8 +404,9 @@ def format_samples(samples)
     metadata_by_sample_id = metadata_multiget(sample_ids)
 
     # Massage data into the right format
-    samples.each_with_index do |sample|
-      job_info = {}
+    # samples.includes(:pipeline_runs, :host_genome, :project, :input_files).each_with_index do |sample|
```

Comment:
why commented out?

Reply:
You might need to refresh. Was doing a test at the timeY